st2common/st2common/services/workflows.py

```diff
@@ -357,7 +357,7 @@ def request_resume(ac_ex_db):
 @retrying.retry(retry_on_exception=wf_exc.retry_on_exceptions)
 def request_cancellation(ac_ex_db):
     wf_ac_ex_id = str(ac_ex_db.id)
-    LOG.info('[%s] Processing cancelation request for workflow.', wf_ac_ex_id)
+    LOG.info('[%s] Processing cancellation request for workflow.', wf_ac_ex_id)
```

Comment:
You don't have to fix the cancellation typos here.  It's unrelated to this patch.